Source/WebCore/platform/BoxSides.h

```diff
@@ -38,13 +38,13 @@ namespace WebCore {
 /* Axes */
 
 enum class LogicalBoxAxis : uint8_t {
-    Inline,
-    Block
+    Inline = 0,
```

Comment:
Doesn't seem helpful.

Reply:
I don't feel strongly, but I made the change to make it more explicit that these values are part of this API, and changing them could affect other code.